src/dvclive/utils.py

```diff
@@ -1,12 +1,14 @@
 import csv
 import json
+import numpy as np
 import os
+import pandas as pd
```

Comment:
Let's import these only as needed inside the function itself. See the example above from `log_image()`.